### DISCUSSION TITLE
module not found jsconfig.js

### DISCUSSION THREAD

**Discussion by vandong1906** [community/community#144278] [Programming Help]
### Body

{
  "compilerOptions": {
    "baseUrl": "./src",
    "paths": {
      "@modules/*": ["modules/*"],
      "@configs/*": ["configs/*"]
    }
  },
}
Error: Cannot find module '@modules/brand/infras/transpot/index' please help me 

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by urmasa-tar** [comment]
Hi and have a nice day!
The link "The module cannot be found" may occur for simple reasons:

Independent fashion Path: Provide preferential fashion paths "@modules/brand/infras/transpot/index". There may be an error in writing the file or folder name. For example, pay attention to the word infras – it is possible, only for a short time, to be similar to both infra and what lilile index claims.js inside the direct transport.
Problems with path settings in tsconfig.json:
See that you have the basic settings for modules in CompilerOptions.paths./modules/*`. In your case, you specified:

"@modules/": ["modules/"],
This means that all references to @modules/* will be replaced with ./modules/*. Perhaps you could find by searching @modules/brand/... what to choose, what to design for your project, and what to do with it.
Links to the downloaded Webpack: If you use Webpack, then you need to make sure that the downloaded Webpack is largely designed to work with other paths. For example:

make a decision: {
alias: {
'@modules': path.resolution(__dirname, 'src/modules'),
'@configs': path.permission(__dirname, 'src/configs')
}
}

I'm not an native speeaker, so I used a translater for that text. Hope the answer is clear anothe

**Reply by vandong1906** [reply]
i will try,thanks you so much

**Comment by mecodeatlas** [comment]
Hi @vandong1906 👋🏾,It looks like @urmasa-tar's response was the answer to your question here. Can you please mark their response as the answer? When you mark a question as an answer, GitHub will highlight the comment and replies to the comment to help visitors quickly find the answer.
![image](https://github.com/user-attachments/assets/b89e7161-c2d1-47e2-a058-242df633e6b2)

**Comment by ravin00** [comment]
reinstall the node modules that will solve the issue.
